docs/src/_reference/command-line-interface.md

```diff
@@ -122,6 +122,10 @@ The `add` command does not run relative to a [Meltano Environment](https://docs.
 
 ## `compile`
 
+<div class="notification is-warning">
+    <p>The compile command is currently experimental, and subject to change without corresponding semantic version updates.</p>
```

Comment:
We could add more copy or just link under the work "experimental".

```suggestion
    <p>The compile command is currently [experimental](https://github.com/meltano/meltano/discussions/7323), and subject to change without corresponding semantic version updates.</p>
```

Reply:
@aaronsteers thoughts on calling it something other than _experimental_? An experiment to me indicates something we have a 50/50 shot on rolling back. Compile is something we'll very likely keep and continue to iterate on. 

Perhaps we could say the structure is not yet stable? Or is there a better phrasing here?

Reply:
@tayloramurphy Thoughts on calling it a "feature preview"? Or we could say that the feature is in beta.

Reply:
@WillDaSilva I guess Beta makes sense - we could also just have a 0.X.X semver on it generally with no promise on when the manifest will be 1.0, right?

Reply:
I don't think we should have versions for features. Let's hope that Meltano itself never becomes so complicated as to warrant that.

Reply:
@WillDaSilva sorry, not for the feature itself, but the schema of the generated file could be versioned right? https://docs.getdbt.com/reference/artifacts/manifest-json dbt-core v1.3 produces schema v7 while 1.4 produces v8. Something like that could happen, right?

Reply:
@tayloramurphy The manifest and `meltano.yml` share a schema and version number. The `meltano.yml` version number is hard-coded as `1`. Ideally we won't add a manifest-specific version number, as that would mean the schema for the manifest and `meltano.yml` differ.

Reply:
Resolved in b38571b

Reply:
Ah ok, thanks for that context.